app.go

```diff
@@ -11,9 +11,12 @@ import (
 	"io/ioutil"
 	"net/http"
 	"net/url"
+	"strconv"
 	"strings"
 	"time"
 
+	"github.com/cezarsa/form"
```

Comment:
yes, we don't download any dependencies during the build (no ``go get`` call) and the test suite passes: https://github.com/ef-ctx/tranor/blob/master/.travis.yml#L18